engine/gamesys/src/gamesys/scripts/script_camera.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020-2024 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+// 
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#include <assert.h>
+#include <stdlib.h>
+
+#include "script_camera.h"
+
+#include "../gamesys.h"
+#include "../gamesys_private.h"
+#include <gamesys/camera_ddf.h>
+
+extern "C"
+{
+    #include <lua/lua.h>
+    #include <lua/lauxlib.h>
+}
+
+namespace dmGameSystem
+{
+    /*# Camera API documentation
+     *
+     * Functions for interacting with camera components.
+     *
+     * @document
+     * @name Camera
+     * @namespace camera
+     */
+
+    #define LIB_NAME "camera"
+
+    /*# makes camera active
+    *
+    * @name camera.acquire_camera_focus
+    * @param url [type:string|hash|url] url of camera component
+    * @examples
+    * ```lua
+    * camera.acquire_camera_focus("/observer#main_camera")
```

Comment:
I think it should be something like `camera.acquire_focus("/observer#main_camera")` to avoid double `camera` mations

Reply:
Fixed